lib/setup/setupSetup.js

```diff
@@ -287,7 +298,7 @@ function Setup(options) {
                 toMaster = null; // new
             }
 
-            let allowMigration = false;
```

Comment:
allowMigration will always be assigned to a boolean some the lines below, Webstorm complaint about an unnecessary assignment